src/App.jsx

```diff
@@ -12,7 +12,8 @@ const App = () => {
    
         <main className="main-content">
           <Routes>
-            { <Route path="/StudentDetails" element={<StudentDetails />} /> }
+            { <Route path="/StudentDetails/:id" element={<StudentDetails />} /> }
+            { <Route path="/StudentDetails/" element={<StudentDetails />} /> }
```

Comment:
For clean and consistent routing, follow this pattern:  

```suggestion  
<Route path="/student/" element={<StudentList />} />  
<Route path="/student/:id" element={<StudentDetails />} />  
```  

This keeps URLs structured and easy to use.